isis/scripts/downloadIsisData

```diff
@@ -146,7 +168,7 @@ def main(mission, dest, cfg_path, ntransfers, kwargs):
     for source in sorted(config_sources, key=lambda x: x.split("_")[-1]):
         parsed_name = source.split("_")
         # If it is a mission, it should be in the format <mission_nam>_<source_type>
-        if len(parsed_name) == 2 and parsed_name[1] in ["usgs:", "naifKernels:"]:
+        if len(parsed_name) == 2 and parsed_name[1] in ["usgs:", "naifKernels:", "pck:"]:
```

Comment:
we would want to check the other types as well. maybe make the dictionary used in the other function global and do like `PATH_MAP.keys()` here. Call the global variable whatever you think is appropriate this is just an example.

Reply:
updated script to check for the source type within a global dictionary `SOURCE_PATH`